canonical-websites/snapcraft.io
# Enable prettier for JS

Fixes #1228 

Adds prettier commit hook, adds prettier checks to eslint, makes sure eslint rules don't conflict with prettier. And updates all JS files to new style.


### Review 
For code review ignore JS files (they were auto formatted), config changes are in commits https://github.com/canonical-websites/snapcraft.io/pull/1082/commits/939872a8ab74ac00afa64a082255ca2cdbad6e28 https://github.com/canonical-websites/snapcraft.io/pull/1082/commits/98f8ddba9cb1eca4b49aec39bc503ecae86a5f93

### QA
- change a JS file with some bad formatting
- ./run exec yarn lint-js
- prettier should complain about formatting 
- commit your changes, files should be modified
- run lint again, it should be no errors
